### ISSUE TITLE
How to use 'save_path' in csv_eval.py ? it seems doesn't work .

### ISSUE THREAD

**Issue by Andrew05200** [yhenon/pytorch-retinanet#207]
I want to save the visual img during eval, and I set it  like this in the train.py  as discribed in `evaluate` func of csv_eval.py (https://github.com/yhenon/pytorch-retinanet/blob/master/retinanet/csv_eval.py).
![image](https://user-images.githubusercontent.com/24999657/105689632-f6981700-5f35-11eb-9727-5da443eb21d1.png)
but, I get nothing in my folder 'val_img'.  thanks

**Comment by AdityaKane2001** [comment]
Yes, that argument is present but not implemented. You may use the visualize.py or visualize_single_image.py modules to get those images. You may slightly have to modify the code as follows:

```     
#in visualize-single_image.py       
cv2.imshow('detections', image_orig)
cv2.waitKey(0)
```

to

```
cv2.imwrite(save_path+image_name+'.jpg',image_orig)
```
